### DISCUSSION TITLE
Autofill + x-model + Safari = No Bueno

### DISCUSSION THREAD

**Discussion by yabdab** [alpinejs/alpine#1862] [7. Bugs (archived)]
Found that autofill does not work when x-model is on inputs using Safari ( mobile AND desktop ).

Here is a GIF showing it happening...
https://p112.p2.n0.cdn.getcloudapp.com/items/12u4XlmZ/eec1fb3f-190e-409d-b912-1e9c88454abe.gif?v=6a587ba54c3779a46a219a45803254e7

And the Codepen  ...
https://codepen.io/yabdab/pen/poPadav

**Comment by yabdab** [comment]
1Password seems to work still. It is only the native Safari autofill that borks.

**Comment by yabdab** [comment]
I would think this is a "show stopper" kind of bug. Surprised it has not been addressed ?

**Comment by MuzafferDede** [comment]
- #1969 should fix this

**Reply by yabdab** [reply]
That is good news! Will test it out when released.

**Reply by MuzafferDede** [reply]
you can test now if you want by this [codepen](https://codepen.io/muzafferdede/pen/KKmjRdb)

**Reply by yabdab** [reply]
Works great! Thanks for fixing  🎉

**Reply by MuzafferDede** [reply]
- #1965 Merged!
